[DNM] msm8937-common: P bringup

JarlPenguin opened the pull request:
Apply device-specific commits (found here: https://github.com/moto-msm8937-common/android_device_motorola_montana/pull/1)

Do NOT merge until p-files.txt is done

erfanoabdi committed:
msm8937-common: symlink /vendor/firmware_mnt to /vendor/f

* For oreo blobs to access firmwares from vendor

Change-Id: I012f02ce967cdd5d3651b3ee175963e244bfea79
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

invisiblek committed:
msm8937-common: move wcnss-service to late_start

* wcnss-service is in charge of copying your wcnss ini from /system to
  /data where framework reads it. If /data is encrypted, or being
  encrypted, this can't happen. Move it to late_start so it tries to run
  after encryption and mount has happened.

Change-Id: I63dce20f84935ac22dd737c9b63b40c69f3e815d

luk1337 committed:
msm8937-common: Add script to update sha1sums of kanged blobs

Change-Id: Id2e8cfb8d3b6f65f69c91a89bacd615fdcc45753

Commit:
data-ipa-cfg-mgr: Don't build ipacm tests on debug builds

Change-Id: I1fd4a4bafc917586ee391fbf6df0da1d12212d36

BadDaemon committed:
msm8937-common: wifi: Enable 2.4GHz channel bonding

* Improve WiFi speed

Change-Id: I06501a8d01d8a802fe5dd7a6c9edfa96c51afbce

weivincewang committed:
msm8937-common: Tune zram performance

* Set page-cluster to 0 as the incremental cost of reading 1 page at a
time is negligible for zram

Change-Id: Ia73d08904551dfc8d57268fc4da3f9f34d10e277
Signed-off-by: Davide Garberi <dade.garberi@gmail.com>

luca020400 committed:
data-ipa-cfg-mgr: Remove verbose cflag

Change-Id: I7d028b4b606f209bd6c27f9f74f4fc04ff69d8c7

luca020400 committed:
data-ipa-cfg-mgr: Disable DEBUG for userdebug builds

Change-Id: Idaee3e6d5b3759028918559b70e04547809bc09b

jacky8hyf committed:
msm8937-common: Add netutils-wrapper to device compatibility matrix.

netutils-wrapper only list a single version x.0 in its
requirement. System manifest might provide a list of
netutils-wrapper versions it supports {x, y, z}.0.

Bug: 64447338
Test: m compatibility_matrix.xml system_manifest.xml -j
      (checks compatibility at build time)

Change-Id: I2ec919fd5378aed7160c636cb7f2bd138db1473f
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

krocard committed:
msm8937-common: audio: Convert audio_effects.conf to XML

Converted .conf format to .xml using:
https://github.com/luk1337/aeffects-conf2xml

Bug: 37492580
Test: dumped the effectFactory state and check it was unchanged
      $ adb shell dumpsys media.audio_flinger | sed '/Clients:/q'
Change-Id: I233a44c2ba0cb4c4d4fb8c3720fc09862ce79827
Merged-In: I233a44c2ba0cb4c4d4fb8c3720fc09862ce79827
Signed-off-by: Kevin Rocard <krocard@google.com>

Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

Signed-off-by: therootlord <igor_cestari@hotmail.com>
Change-Id: I7bf4b5f0a4bdba1fd0595733a69bc810a55758f7

rpius committed:
manifest: Uprev to supplicant 1.1 & vendor HAL 1.2

Bug: 65673412
Bug: 65647646
Test: Compiles
Change-Id: I18b102dbc5cd85b66dea392cea4d103628852427

Commit:
msm8937-common: ueventd.qcom.rc: set permissions of /dev/qce device

To support end-to-end DRM content playback with SG List feature,
android.hardware.drm@1.0-service needs to have access permission
to /dev/qce to map & unmap ION buffer virtual address with HW Crypto
Engine. So, set /dev/qce device permission 0660 and let it be
accessible by user "system" and group "drmrpc".

Change-Id: If9c0ed70acc2bb063344692374f51441fe84eff6
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

Commit:
msm8937-common: init: Move mkdir /data/nfc/... to rootdir init.rc

These are directories used by the system so they should be created by
the system.

Test: treehugger
Change-Id: I4c2d381423e002705f4de09d34a63a49ec473688

Signed-off-by: Alberto97 <albertop2197@gmail.com>
Change-Id: Ic023380dc7e0517e23be739dd041760d7b07143d

rpius committed:
manifest: Add hostapd HIDL interface

Bug: 36646171
Test: Manual softap toggle
Change-Id: If3c9ea38317b1fee66a383ad88cabf695748853a

Signed-off-by: Alberto97 <albertop2197@gmail.com>

luk1337 committed:
msm8937-common: Handle moved blobs in update-sha1sums.py script

Change-Id: Iccab8c8620c7ff680768aac2d9cbf5b5067af8d7

bgcngm committed:
data-ipa-cfg-mgr: Disable all the debugging

 * Same logic as in commit 5463a591007ae63ba3720ed295eac5ef19cc2ce0.

Change-Id: I7ca76b868a4678fb5d8280928f90160973e51871

Commit:
msm8937-common: Dynamics Processing Effect

Dynamics Processing configuration for marlin

Bug: 64161702
Bug: 38266419

Test: manual testing
Change-Id: I0b0d55af3774d798af92159b98764d875a4b7628
Signed-off-by: Alberto97 <albertop2197@gmail.com>
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

bgcngm committed:
msm8937-common: Add unpinning support to update-sha1sums.py script

Change-Id: I9035b449cea6d78927a459dc247075b3d4512e46

Commit:
msm8937-common: Start loc_launcher as gps user

Start loc_launcher as gps user and set uid/gid of loc_launcher
to gps and enable SETUID and SETGID caps for loc_launcher, so
that we can start loc_launcher as gps user instead of root and
still have capabilities for SETUID and SETGID. Also remove not
required permissions groups for loc_launcher.

Change-Id: Ie9f1506874b3c3148f9170c8d5db9afd1e70025c
CRs-Fixed: 2192881

krocard committed:
msm8937-common: Migrate to audio V4

This is needed for the microphone characteristic API support.

Test: regression test on the board
Bug: 38184704
Change-Id: Ife559125850277865451f92ea862e7ec47fc9da7
Merged-In: Ife559125850277865451f92ea862e7ec47fc9da7
Signed-off-by: Kevin Rocard <krocard@google.com>
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

Commit:
msm8937-common: Renamed config_dozeAfterScreenOff

New name is config_dozeAfterScreenOffByDefault
Bug: 73178121

Change-Id: Ie3d1ea5a2cc480655592bf6e49b37f5ea07f3f58

Signed-off-by: Alberto97 <albertop2197@gmail.com>
Change-Id: Ic340a4d1719bf361a31d85e2842f668a82eb18d9

weivincewang committed:
msm8937-common: Wait for hwservicemanager ready before running fsck

As property service is part of init, and if init is busy with built-in
command e.g. fsck, the propety_set() won't go through until the built-in
command is finished.
Move the start of hwservicemanager earlier and wait for its done after
launches bootanim and its dependencies (there is not wait time during
boot but the command is as a safeguard).

Bug: 74437368
Test: boot with simulated long fsck and bootanim shown early
Test: boot see 'start_waiting_for_property("hwservicemanager.ready", "true"): already set'
Change-Id: Iab4b857a51e0e6b8b059d063c1f0d65a0e987622

critsec committed:
msm8937-common: Advertise Vulkan 1.1 system feature

Bug: 66875961
Test: atest VulkanFeaturesTest
Change-Id: I58f904984320dee174637ba0ddfecdd343fab7e6

Rashed97 committed:
msm8937-common: Switch to BOARD_VNDK_VERSION=current

Change-Id: Id4d4424405ce9f088132fc74c47ef7574d6e1e0b
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

Commit:
msm8937-common: change ownership and permissions of sensors device node

Change the user and group ownership and permissions of sensors
device node.

Change-Id: I251af01d4f0204c784667fe92b3998c197f75d8e
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

Commit:
msm8937-common: Do not create fingerprint directories in init

Add changes to not create fingerprint directories in
init.qcom.rc. It will be done in fingerprint module
init script.

Change-Id: If199cf875bb4201fcda923b0e8162b06d8d93d1a

Commit:
msm8937-common: Moving persist mount point to /mnt/vendor/persist

Apply required user, group and permissions once persist
folder is created through __mount and parititon is mounted.

Run restorecon on /mnt/vendor/persist.

Change-Id: I7f54b478de898491e32560d5a37657ac370bf893

Vachounet committed:
msm8937-common: start that treble disaster

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ib353ff5101fa1ecc06b88be9947ab01d11a1fad7

Vachounet committed:
msm8937-common: move firmware symlinks to vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Iabc6ee369b42c93b99a6190b9cf26cdb79dd0164

Vachounet committed:
msm8937-common: split vendor and system specific props

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I408e25050ef6eddce4988e3f9c9f83ccb8643266

Vachounet committed:
msm8937-common: move firmware to vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Idcb9f0880c90203055154d1425eb9473e3498027

Vachounet committed:
msm8937-common: set vbmeta-version in compatibility_matrix

* Fix vendor image mismatch warning on boot

https://github.com/LineageOS/android_frameworks_base/blob/lineage-15.1/core/java/android/os/Build.java#L899

Commit:
manifest: Changing healthd version to 2.0

Change-Id: I13ddaa133bb9edcca200d5fe7320cf5fa9f66f0f

mikeNG committed:
data-ipa-cfg-mgr: Disable NAT_DUMP on non-debug builds

* Fix compilation with disabled debugging

Change-Id: I7483735fb935798b57e0072bb14ed8a476857786

TheScarastic committed:
msm8937-common: fix BT MAC address reading

* kubersharma001: tweak it for new bt hals

Change-Id: I82e31981d91c8e1999dbb49b3d40eb4248f01f50

Commit:
configstore: uprev configstore (1.0 -> 1.1)

Bug: 69691076
Test: built and tested on sailfish
Change-Id: Ieb4b329ccf2f56e304cd2b62ba12183e514aaff1

Vachounet committed:
msm8937-common: switch to xml audio policy

Change-Id: I62d03ba6caa1fe0039a0118d50bdbbd482040ee8

Vachounet committed:
msm8937-common: copy keylayouts to vendor

* And update fingerprint one

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I12370d46b778fa8137654f43c3100d86a1297937

Vachounet committed:
msm8937-common: audio: add back FM entries

* Gone when switching to xml policy

weivincewang committed:
msm8937-common: start bootanim before mounting data partition

Bug: 30118894

Signed off by: Nolen Johnson <johnsonnolen@gmail.com>

Change-Id: If84764e3ce51f467b2dfa867ce28e1e396422d45
Signed-off-by: Davide Garberi <dade.garberi@gmail.com>

olvaffe committed:
msm8937-common: Start hwservicemanager early

Test: builds and boots
Bug: 35173320

Change-Id: I9f060904174103a6a145a83cbb4c100728ff6df6

weivincewang committed:
msm8937-common: Make sure graphics HAL services started before mounting data

Bug: 36409349

Signed off by: Nolen Johnson <johnsonnolen@gmail.com>

Change-Id: I1df0bd7878c7d67d3695ad1644c6b928cee0e323
Signed-off-by: Davide Garberi <dade.garberi@gmail.com>

weivincewang committed:
msm8937-common: skip wait for hwservicemanager

Wait for hwservicemanager is no longer needed
after this CL:
https://android-review.googlesource.com/#/c/356043/

Bug: 36399429
Test: nash boot

Signed off by: Nolen Johnson <johnsonnolen@gmail.com>

Change-Id: I12c5adab70dc519b5e834eb35262ff73d214978b
Signed-off-by: Davide Garberi <dade.garberi@gmail.com>

Commit:
msm8937-common: Swap disk creation

Create swap disks

CRs-Fixed: 2264866

Change-Id: Id57a4e8d413c7dafb49d073980f10010003f616e

erfanoabdi committed:
msm8937-common: qsap_shim: Fix for VNDK_VERSION=current

Change-Id: I8e5ec3068914e085d4adac178a47421a1c515a89
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

Commit:
msm8937-common: Cleaning up Impl tag

Change-Id: Ieae8dc4c22106e345b0d21b57e42e398495d5d7c

Commit:
msm8937-common: modified manifest entries to allow drm 1.1

Change-Id: Ide4f9a1ecff3a89b1bf92e04a6292e2511e06ec9

Commit:
msm8937-common: IMS: Whitelist org.codeaurora.ims package

org.codeaurora.ims has become a priv-app
to acquire some permissions we need to
whitelist the package.

Change-Id: I3f31f949e054cc79f8cff99e66cbcb70f5f0715f

Commit:
msm8937-common: Set partitions for loop device.

AdoptableHostTest in CTS uses virtual disk feature for testing.
This change is to enable partitions for virtual disk.

Change-Id: Ic89de587b2c0448fd55b55bf19086e92e843a923

Commit:
msm8937-common: ueventd: init: add permission entries for location component

Add permissions for location components in init.qcom.rc and
ueventd.qcom.rc

Change-Id: I07f3b57470336165393a68144faccf26cdbca98c
CRs-Fixed: 2299356

Commit:
msm8937-common: set rlimit memlock to 64MB

without this, max locked memory (RLIMIT_MEMLOCK) is being set
to 65536. Traffic controller fails to start as a result since
netd requests for pages (=255) more than that of the available
pages (= 65536 >> 12 = 16)

Change-Id: I746c1e2b3cb01c7af8e09987686dfb5190aa7cd3
CRs-Fixed: 2292581
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

AdrianDC committed:
msm8937-common: init: Resolve vendor HALs for early bootanimation start

 * Adapt the service from configstore-hal-1-0 service
    to vendor.configstore-hal for AOSP P

 * Resolves surfaceflinger first launch crash
    due to configstore not being running earlier enough

Change-Id: I5aae6d8357ce71933e7c6673e9e4d37adb55a6af

GtrCraft committed:
msm8937-common: Update libsensorndkbridge

from potter

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If7dac79ce5c314963f8b17dee04b708625b71321

Commit:
Add fqname 1.0::IGnss/gnss_vendor

ILocHidl is derived from IGnss, so a ILocHidl service
named gnss_vendor is also a IGnss service with the same name.
CRs-fixed: 2290565

Change-Id: I387184b071825374d32d22c5532eccaa5bacb8a5

Ziyann committed:
msm8937-common: camera: Specify internal video devices to ignore by external provider

The external camera provider occupies our camera V4L2 nodes, potentially
crashing the kernel driver and blocking the camera HAL.
Unfortunately, there's no easy way to just disable it, so let's just tell
it to ignore the internal video devices.

Change-Id: I00c67de1b94a1f7eca1bf458d1d0e4de3507aa81
Signed-off-by: Daniel Jarai <jaraidaniel@gmail.com>

GtrCraft committed:
msm8937-common: Update ims & gps blobs

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I37a95b0c29bede481d2845d432ea6a81234d0e6c

Commit:
msm8937-common: ueventd: add permission for gps component to read pps device

GPS components make use of DR_SYNC pulse generated by GNSS
hardware, which was received through pps device, hence
require permission for reading this device

Change-Id: I00bd3b8aa88059fb94338b6ba1d107d77c06a5e0
CRs-Fixed: 2299356

therootlord committed:
msm8937-common: update thermal-engine

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ifca43b931f46b854f71a5fc2ff564fc12dc7aca6

therootlord committed:
msm8937-common: update esepowermanager blobs to P

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ife245a2cd5e97a8e6283b8a88cb5f3bbf9fd849a

therootlord committed:
msm8937-common: update FM hals to P

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Idd6f93812c7be9ce910c356733ae71865a029ca4

therootlord committed:
msm8937-common: update GPS blobs to P

from potter

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I4fd6fd7171d88383757a71d65ef6262fde4ab2ed

rahulsnair committed:
msm8937-common: config.fs: system/vendor -> vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I9c76992144826b36d1407b6f8f6e081964d3b108

therootlord committed:
msm8937-common: switch to 64b pm-service

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I14163b2a390129a8a11a224b230e54eaf1d6f811

therootlord committed:
msm8937-common: config.fs: add missing AID_VENDOR_QCOM_DIAG

makorn645 committed:
msm8937-common: update display stack from Nokia 7 Plus (Onyx) WW_3_190

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I3c1b5ce5a1b82a45d488d9e88e46737d57d2e22b

therootlord committed:
msm8937-common: remove audio_output_policy.conf

* We're using audio_policy_configuration.xml

Change-Id: I20dbc70ecdcbec47c29836cb1c570b86ca10654a

moetayuko committed:
msm8937-common: Copy vendor HIDL interface libs to /vendor

Change-Id: Ia95e09b21ce42137cac9a1dd22f208fd9de9bf20

bgcngm committed:
data-ipa-cfg-mgr: Use generated kernel headers

Change-Id: Ie33f0b8b7996536a73f660cba5b4700964d8b03f

Commit:
msm8937-common: init: Changes to stop rtp daemon during shutdown

* Changes to stop rtp daemon during shutdown

Change-Id: I08d56dba615136a2c02c3aa11a55bd3e7d84f6a0
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

moetayuko committed:
msm8937-common: Update qti_whitelist from LA.UM.7.5.r1-03100-8x96.0

Change-Id: I93ad3de87cbb78d2155fd0b4304738c2080554f5
Signed-off-by: Ashwin R C <ashwin2001achu@gmail.com>

therootlord committed:
msm8937-common: update thermal-engine from LA.UM.7.6.r1-02500-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I2d79ad989da86dd6cc5c592574c896869daf6ead

therootlord committed:
msm8937-common: update for the new mount points

Change-Id: I988e1ab25565a37ffa02b103a21438967deebca2

therootlord committed:
msm8937-common: update sec_config from LA.UM.7.6.r1-02500-89xx.0

Change-Id: I43ae89d1203495d15694fe83ef04c08e1284bd54

Commit:
msm8937-common: Enabling USB enumeration in power-off charging mode

sys.usb.configfs can't be set from .sh file as per new Android
sepolicy rules. Hence set sys.usb.configfs in target rc file based on
ro.boot.usbconfigfs property for USB to work properly in
power-off charging mode.

Change-Id: I075c1d33e4018099e3deca64e8b2737f1a0c1d4b
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

therootlord committed:
msm8937-common: update esepowermanager blobs from LA.UM.7.6.r1-02500-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ieb1cd403e7f85d1599d0bb18d53a3dd84a78dd0b

therootlord committed:
msm8937-common: update AptX blobs from LA.UM.7.6.r1-02500-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I842d1c77b36224b3dca7213b6679a2043f0f5b98

therootlord committed:
msm8937-common: update bt blobs from LA.UM.7.6.r1-02500-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ia0b5210f9db1742167072e038748c074c37d714a

therootlord committed:
msm8937-common: update FM hals from LA.UM.7.6.r1-02500-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I245ff567904e7973f56420aa7f815945e869af98

justks committed:
msm8937-common: update CarrierConfig overlays from LA.UM.7.2.r1-05400-sdm660.0

Change-Id: Ib23f2f386c6e43467a114c488f2f06bb53b602a6
Signed-off-by: Ashwin R C <ashwin2001achu@gmail.com>

moetayuko committed:
msm8937-common: Kang ims.apk and qti-telephony-common.jar from daisy

* From: xiaomi/daisy/daisy_sprout:9/PKQ1.180917.001/V10.0.1.0.PDLMIFJ:user/release-keys

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If134ee58c23f7c04bc387bb45cd9bcc81a5affe3

moetayuko committed:
msm8937-common: Set new QSEE listener status flag on old one changed

Change-Id: I61ebbbaa7c13009d537f9ee9177da9dbf9b0fec0
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

intervigilium committed:
msm8937-common: Use local DATA_IPACFG_MGR

Change-Id: I6b074707510bde8fb001b42dbbe1654c6ebdb6ad

justks committed:
msm8937-common: kang vulkan blobs from addison

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ied5247746e2175dea40f4f3c52a878a74faf4b62

therootlord committed:
msm8937-common: update GPS blobs from deen PPK29.68-16-8

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ic86a3d2d3e4e313ceba1aede98d9eb93a8a90597

mdeejay committed:
msm8937-common: Add WFD blobs from daisy

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ib4aefe34ab5f92f9cd493eee929c0ea98429c019

therootlord committed:
msm8937-common: rootdir: ueventd: partial update from LA.UM.7.6.r1-03700-89xx.0

Change-Id: I159973d4c169314653c36a2d8dd12653447f4fc7

therootlord committed:
msm8937-common: update esepowermanager blobs from LA.UM.7.6.r1-03700-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Iced0085afeb641e17f64a10a385bd914a0016310

justks committed:
msm8937-common: kang libllvm-qgl.so from addison

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ic079549aac568ec9b0a9b99121c9f1c79dbd55e3

Alberto97 committed:
overlay: set minimum brightness

Change-Id: Iaecb9a2e5e2588f5bbe3e1d7e3d8df3fc01d7550

vladimiroltean committed:
msm8937-common: introduce kang mode

Also do the following:
* Fix passing of arguments longer than one word, such as --section "Hello world"

Wight554:
* Small cosmetic changes
* Fix some shellcheck warnings

Change-Id: I917ed382a8c0e33ba7709151f2743a723dc50617
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>

nichcream committed:
msm8937-common: Update perf stack to Pie

Change-Id: Icd27ca77fd13fd8dbbb9f3b1b61e19448fb88e1d

therootlord committed:
msm8937-common: gps: add missing config file from deen

Change-Id: Ia5643db798f6dd82bf411655405e53d17046be82

justks committed:
msm8937-common: import data-ipa-cfg-mgr from LA.UM.7.6.2.r1-04800-89xx.0

Change-Id: I6a942a0efc1af0ea545f05279cef3056a3c953a6

justks committed:
msm8937-common: build IPACM

Change-Id: Iad6252824bcef4528607dcf91264c1560d5f0756

Vachounet committed:
msm8937-common: update sepolicy

Change-Id: I1218c2f96bfbff728305100080e3cbdf6414f19a

Vachounet committed:
msm8937-common: update privapp perms

Change-Id: If6773d00111f12763f26da2b94450ab8d9a0ccfc

aviraxp committed:
msm8937-common: Configure number of physical SIM slots

* Needed by telephony-common

Change-Id: I18c058d295ed2eaf23e8ca5d1f3933b65806ed48

Vachounet committed:
msm8937-common: address new denials

Change-Id: I2d46992f18907f17b04be49aed2f0f426686a903

Vachounet committed:
msm8937-common: update ACDB blobs

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Iec1eb0a753197df4ead0ccd03e37efd5dfe45b4f

GtrCraft committed:
msm8937-common: Set swappiness back to 100

* Google seems to have changed zram behaviour (much less agressive)
* Also follow big G's way for enabling zram

Change-Id: I40758b75fecd9df2b11a35199cd3594b3224a3c7

GtrCraft committed:
msm8937-common: init: set custom zram sizes for different amounts of RAM

* We have 2/3/4gb ram variants of montana, they all need a different amount of zram
* Also add ro.boot.ram to detect right amount of ram, credits to @Vachounet

Change-Id: I35c1e00722c6f9753f1d45f0ecbad54d9a0c368b

Vachounet committed:
msm8937-common: address new denials

Change-Id: Idfa8f3c411f7fc41696379a26d28ca498d9140a5

Vachounet committed:
msm8937-common: Update keystore and gatekeeper HALs

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ie87554d92c4f2bc5ee9de03f4b49d1cf2644846d

Vachounet committed:
msm8937-common: add qti_libpermissions for qcrilmsgtunnel

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I526b5693ee527c381bfb3efda3151a5fc6b0e9d6

justks committed:
msm8937-common: update ims.apk which passes ims status for us

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I8f6eae470d3ebced66ecca101a1798d11b11e597

mikeNG committed:
msm8937-common: Update SDM blobs from LA.UM.7.5.r1-03700-8x96.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ie3e1ed1fda517946593b7917c21b78d5a9ba9523

bvlj committed:
msm8937-common: Build Trust HAL

Change-Id: I92706987f8d3094cdfb82e71e5331b7efe69e5df
Signed-off-by: Joey <joey@lineageos.org>

bgcngm committed:
msm8937-common: Replace KeyDisabler class with Touch HAL

Change-Id: I7b87ae0ad834ba02a78696afe393d9d4f8920fbd

moetayuko committed:
msm8937-common: Wait for MPCTL to start on boot

Change-Id: Ib79b48e1d362872d9d97373d5ca2ad42f7fbc638
Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

therootlord committed:
msm8937-common: update time service blobs from LA.UM.7.6.r1-03700-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ic4a0a5ec56b8ed8f2a9caf54d8f3a4ad8465d5c8

bgcngm committed:
msm8937-common: Migrate to livedisplay 2.0

Change-Id: I513c284c534ce3abc27bf374513280171454f4d1

Vachounet committed:
msm8937-common: switch to binderized vibrator service

Change-Id: I24cc8998aa0b38e090fd40515795bb4f313b081b

Vachounet committed:
msm8937-common: switch to binderized light service

Change-Id: I74e06a2c112d9a447ab86c50c13fab69687b4454

Vachounet committed:
msm8937-common: kang gemini health hal

Change-Id: I619b9e96dad69f24f373dd203b4ee1cf2248a824

Vachounet committed:
msm8937-common: update manifest

* Cleanup
* Add missing hals
* Sync with CAF

Change-Id: I36bf8c3a349a8c8734de81e64159329fe321ec2f

Vachounet committed:
msm8937-common: sync init.qcom.rc with last CAF changes

Change-Id: I0c06e086d3843faafa6787dc69b7f7541dc3304f

Vachounet committed:
msm8937-common: set TARGET_ENABLE_MEDIADRM_64

Change-Id: I11d59ce11ac49ff552f33d2342d636c1c0cf3d75

Vachounet committed:
msm8937-common: address some wcnss_service denials

Change-Id: Id251ca6e5bea7c88bf2da41a1cebb89fcc9dcbaa

Vachounet committed:
msm8937-common: update blobs for libperfmgr

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I4212666334e7fb037aadd0a051a342a083ce859c

Vachounet committed:
msm8937-common: add QTI wifi hals

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ifcf5de4b34b04a9bf118768b3fbbe01d33049b57

Vachounet committed:
msm8937-common: add back FM support

Change-Id: I86fd0251a1278cc7a5d0102ec554979a3732e4fe

Vachounet committed:
msm8937-common: don't break hotspot again

Change-Id: I341b29eec7177bc0bfc6e0c513ec4398df395e2e

Vachounet committed:
msm8937-common: address some denials

Change-Id: I0aad0e56ff4ed9488476aa77488f7b7d54b44e8f

coldhans committed:
msm8937-common: Adjust health hal namespaces for our device

* Thanks to @Ashwin4RC for noticing this

Change-Id: Ic528912eddce90e890f57ed546da45a15fa70bb2

Vachounet committed:
msm8937-common: update DRM blobs

* from ocean

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If6647f8ee26b2848cfc400326720a0f48d4a1138

justks committed:
msm8937-common: init: remove energy-awareness

Change-Id: I402bd3441eddafbcb06d6971b975f969669087b8

justks committed:
msm8937-common: remove energy-awareness

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I30fd68410981e0f5b3ddde116b631ba1fa1c3b69

justks committed:
msm8937-common: WCNSS_qcom_cfg.ini from deen's pie release

Change-Id: Iab2583eb89d3d9620151b455f068585bcaa55696

erfanoabdi committed:
msm8937-common: disable runtime linker namespace isolation

Change-Id: If272803d3665b5aaed5550aa371244e0b73c19d6
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

therootlord committed:
msm8937-common: cleanup nfc permissions

Change-Id: Ifb47d4845ce0c30b9d3f481e31147d4f841642b1

therootlord committed:
msm8937-common: update p2p_supplicant_overlay from LA.UM.7.6.r1-03700-89xx.0

Change-Id: I8c417c6092635b7226bb1dde4793561c1c3b038f

mikeNG committed:
msm8937-common: init: Remove ACL/CABC/HBM sysfs permission changes

* all are already set in livedisplay rc

Change-Id: I6e2f7a5d4472d4a687272c1cf696250be00a3723

javelinanddart committed:
msm8937-common: Build livedisplay sysfs service

* Add supported interfaces to manifest while we're at it

Change-Id: Iaad13fcdfb603d144599133339f9c0c6a0d1c66a

mikeNG committed:
msm8937-common: Stop copying WCNSS_qcom_cfg.ini to userdata

Change-Id: I7b9fbc0e26a1563f3e13fe0f9d58308b618a5702

KalilDev committed:
msm8937-common: Drop TARGET_LD_SHIMS (1/2)

* Use patchelf instead, to avoid linker hacks

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: Ief170e0a9a393dbcf4ed0ced432c80cacfcec94d

KalilDev committed:
msm8937-common: Import aljeter pie media configs

Change-Id: I58a70f661db86c25fdf38c6447bc0958a9691446

KalilDev committed:
msm8937-common: Ship v27 libui (1/2)

* Needed by lib-imsvideocodec.so

Change-Id: I291da988a6c92ed32f5d38366393359401022826

elektroschmock committed:
msm8937-common: Remove WITH_LINEAGE_CHARGER

* Lineage Charger is opt-in now.

Change-Id: I8cee5b4acb9544014ee2b7d6a91ee963ce9226dd

npjohnson committed:
msm8937-common: Remove deprecated TARGET_USES_64_BIT_BINDER

Change-Id: Ibafc06659dae21e8acdd132bc2de7478008682b0

erfanoabdi committed:
msm8937-common: Import libbinder and libcutils from griffin

Ashwin4RC committed:
msm8937-common: Add vendor prefix to gralloc-2-0

* init: Command 'start gralloc-2-0' action=late-fs (/vendor/etc/init/hw/init.qcom.rc:55) took 0ms and failed: service gralloc-2-0 not found

Signed-off-by: Ashwin R C <ashwin2001achu@gmail.com>
Change-Id: Ic444c7c683ab94b9a37e7d100cde19485db651b9

erfanoabdi committed:
msm8937-common: build vendor version of libbson

Change-Id: I1c8f7e11529016ec04e52586f6aba0cfa02e4733
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>
Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

bgcngm committed:
msm8937-common: Move to common data-ipa-cfg-mgr

* It is now available at vendor/qcom/opensource/data-ipa-cfg-mgr

Change-Id: If66808c4bc2c02f7d253bdc1f7b34b83f559a047

Sohamlad7 committed:
[TEMP] msm8937-common: move wcnss_service to vendor

Change-Id: I704cbc8b3a58ad59636acfa018fcf9fe41ecda1d

Sohamlad7 committed:
msm8937-common: overlay: show navbar

Change-Id: I570403ce6457b72978d0d698006ee8e266147684

Sohamlad7 committed:
msm8937-common: init: Add 'vendor.' prefix to fm init

Change-Id: I1fd0aa2ecf90bd32a9cf70114642654e65b6f383

Sohamlad7 committed:
msm8937-common: remove display mode for now

Change-Id: I177eeebd298d505fdae680e1bc52c56f8f0edff3

justks committed:
msm8937-common: update libOmxVideoDSMode.so from daisy

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I18f41d5cdf0ceb7c5cf454a7c9decf729ca55f78

justks committed:
msm8937-common: update init.qcom.rc

 * create needed dir
 * remove irq

Change-Id: I04d89f623142940fb4ff1dee96b8e6439c129e9a

justks committed:
msm8937-common: import public.libraries.txt

Change-Id: If53c7fe2b601f9779fc94cb2fd7187d9ccca7b79

justks committed:
msm8937-common: sepolicy: cleanup

 * remove msm irqbalance since its kinda useless and causes dac_override

Change-Id: I5d932b6cb3fcb475eb19b32de989a9e8e296ade8

justks committed:
msm8937-common: add more wfd props

Change-Id: I21d931f5cf6cc7d4074d317881a1ca981b24228f

AsdMonio committed:
init: Remove duplicate (time service directory creation)

Change-Id: I3540e67001a29cf11fd6bb17082a3148ae64ef41

AsdMonio committed:
msm8937-common: Enable Wi-Fi calling

Change-Id: I592a4be1bc55ae0f442b6591cfe70051305fb3ca

AsdMonio committed:
libshims: Move adsp shim to vendor

Change-Id: I22475a23fbb0f74993d4f57648c858c762b0f21c

mvaisakh committed:
msm8937-common: libshims: Set correct header dependency

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>
Change-Id: Idd7c1a26141299eecca716868c571cbc2ef6f6f3

justks committed:
msm8937-common: system.prop: re-add hal1 packagelist

https://github.com/LineageOS/android_frameworks_base/commit/fbb37381234dd3e93962d30852917b903c6d5daf#diff-925034363271b2b3a8a0750a8c76504e

Change-Id: I07f72564d711fb6218173ce78f43867cf1015ee9

justks committed:
[DNM] msm8937-common: force hal1 on only whatsapp and instagram

Change-Id: I93f31a70dde0d5787f2ca53f638ee189023baca2

neobuddy89 committed:
msm8937-common: rootdir: Remove cpuset foreground boost

Cpuset foreground boost is departed since
https://android.googlesource.com/platform/frameworks/base/+/a712d4058f6f85268838

Change-Id: I65216f3e9091ce38aab8d58d261ba7a695ba80a4
Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

Sohamlad7 committed:
msm8937-common: remove msm_irqbalance

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I695ecce609d99817b28f8fe926f0e5d021e5cbf9

mvaisakh committed:
msm8937-common: libshims: Update ADSP Shim

* updated from https://android.googlesource.com/platform/external/tinyalsa/+/refs/tags/android-9.0.0_r35/mixer.c

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>
Change-Id: Icc845331b6cb343420a6813a9c8dbade0c8070a5

bgcngm committed:
msm8937-common: build Touch HAL to enable KeyDisabler

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: Iad89ce788691d3b69590df28e00262510006533e

mvaisakh committed:
msm8937-common: audio: Add audio_tuning_mixer from OSS Hal

* From CAF Tag LA.UM.7.6.2.r1-08300-89xx.0

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>
Change-Id: Ie32335b83f9203cadc2b02f6ff02b5f4a0612a4e

Vachounet committed:
msm8937-common: copy missing vendor.display.config

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I5aa44380ddc385cb09cecc74d5fcfaca052bc5ce

Flex1911 committed:
msm8937-common: Add QCOM's WFD implementation

* Sadly, out-of-the-box Miracast support was killed in P, so
  we need this to keep Wi-Fi display feature working.

Change-Id: I3dfe53dd3c73d9777e0bddd18a48c0eb9c91a139

AsdMonio committed:
msm8937-common: Move more libraries to vendor

JarlPenguin committed:
msm8937-common: touch: fix build on BOARD_VNDK_VERSION=current

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: Ie24e7002be21e43849c24fb34734905aa609b510

Sohamlad7 committed:
msm8937-common: hardware-moto-sensors: fix build on BOARD_VNDK_VERSION=current

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I5100acb19f8dfd885c35641176eeca95d357a73f

AsdMonio committed:
msm8937-common: Import SDK 27 libbinder

Change-Id: I47307cc503ce6f00fd673dd5145a370b2d69a0cb

YumeMichi committed:
msm8937-common: Regenerate compatibility_matrix.xml

 * ./out/host/linux-x86/bin/assemble_vintf -m -i system/libhidl/vintfdata/manifest.xml -o device/motorola/montana/configs/compatibility_matrix.xml

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If97ff63e64c7d061bc11811f520c767dee52e2ee

erfanoabdi committed:
msm8937-common: Set target FCM version = 2

Change-Id: Ife569e562af48a35fdc48c01b228e6a877bb7f12
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

JarlPenguin committed:
msm8937-common: thermal-engine: sync with LA.UM.7.6.r1-05300-89xx.0

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: I9b981bdea29bb3786ab3c3773b8e87f2e44557fc

Vachounet committed:
msm8937-common: add sku string to device model

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I696f5a8a08699018d0015ac6147d5e469f19f3ac

mvaisakh committed:
msm8937-common: update thermal HAL from CAF

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I1f82a520d2046ace1cb09ae71768ed5650dba15f

JarlPenguin committed:
msm8937-common: vndk-sp: sync with LA.UM.7.6.2.r1-09100-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I7788dcb844c789fe766fdf22db2a83eb1dedd213

Alberto97 committed:
msm8937-common: convert libhidl and some shims to blueprint

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Iab4b260e36a342821dc050d27aef66140e346afc

JarlPenguin committed:
msm8937-common: proprietary --> vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I556aaf50f0fef4b1536b962d69f86fab27833e22

JarlPenguin committed:
msm8937-common: update sec_config from LA.UM.7.6.2.r1-09100-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Id49496dc3383ed57a8b7950667d2e22fd92409f8

JarlPenguin committed:
msm8937-common: update qti_whitelist from LA.UM.7.6.2.r1-09100-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I69551e3b799cabf668c4ebeefd9bb51f6e109c6e

SLudkeG committed:
msm8937-common: hardware-moto-sensors: Move sensorhub firmware path to /vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ie6879ba6c3c9d85cb7ab9f58dd242d9235bc0442

JarlPenguin committed:
msm8937-common: build vendor version of HIDL blobs

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: Ia22659217b46d96f332db1f922056b05074c706b

JarlPenguin committed:
msm8937-common: mount /fsg in /vendor

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ic2c617f14bf9d19d00addde04f98076e05728fde

Fagyi committed:
thermal-engine: start as class hal

thermal-engine serves the IThermal 1.1 HAL.  Start as class "hal" to re-register with hwservicemanager if hwservicemanager dies.

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

neobuddy89 committed:
msm8937-common: Enable wifi mac randomization

Change-Id: I945f82f761e77c90c8c11aa9e5b9ee1f9296f4a7

neobuddy89 committed:
msm8937-common: enable aosp user preference option

UtsavBalar1231 committed:
msm8937-common: overlay: Ignore RSSNR signal level

Improve reception

Signed-off-by: EsromTech <esromgalaxy5@gmail.com>
Change-Id: I25590a4eca0b0b350b3dac0a006804212381dd69

JarlPenguin committed:
msm8937-common: import and build recovery init scripts

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: I0d56e58b0fd1802a0799f7744f1e67fc6051abca

AsdMonio committed:
msm8937-common: drop gpu mapper shim

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

mvaisakh committed:
msm8937-common: thermal-engine: sync with LA.UM.7.6.2.r1-09500-89xx.0

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

mvaisakh committed:
msm8937-common: thermal: sync with LA.UM.7.6.2.r1-09500-89xx.0

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

mvaisakh committed:
msm8937-common: uprev Thermal hals to 1.1

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

zeelog committed:
msm8937-common: Update gfx (postprocessing) from AquariusX2

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

erfanoabdi committed:
msm8937-common: Setup device soong namespace

Change-Id: I2e9c0ab68a0fe86ba18e3955e2823b87d88e835a
Signed-off-by: Erfan Abdi <erfangplus@gmail.com>

bgcngm committed:
msm8937-common: Define Soong namespaces in the common product makefile

This is where it belongs.

Change-Id: Ibea0e8fe5dbc26d8953a9735f0b38cf087805b76

mvaisakh committed:
msm8937-common: Disable thermal-engine header export

Signed-off-by: Vaisakh Murali <vaisakhmurali@gmail.com>

Commit:
msm8937-common: New ION API flag is deprecated

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I7f917746219c84eee6c348fd12aa197d766b8111

JarlPenguin committed:
msm8937-common: get rid of extra hw props

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: thermal: cleanup

* Remove unused sdm845 and msm8998 files

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: build android.hardware.audio.effects@2.0-service

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin commented:
+1, seems good to me, still needs testing

JarlPenguin commented:
Cherry picked more commits from https://github.com/mvaisakh/platform_device_motorola_cedric

wiktorek140 commented on BoardConfigCommon.mk:
```diff
@@ -67,8 +65,8 @@ AUDIO_FEATURE_ENABLED_WMA_OFFLOAD := true
-BOARD_USES_GENERIC_AUDIO := true
-TARGET_USES_QCOM_MM_AUDIO := true
+USE_CUSTOM_AUDIO_POLICY := 1
+USE_XML_AUDIO_POLICY_CONF := 1
```
Should be 1? not true?

JarlPenguin replied:
This is correct, since XML audio policy gets rid of those flags

wiktorek140 commented on BoardConfigCommon.mk:
```diff
@@ -117,17 +116,20 @@ endif
 
 # Kernel
-BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=30 msm_rtb.filter=0x237
+BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=30 msm_rtb.filter=0x237  androidboot.usbconfigfs=false
```
additional space, probably it don't change anything

JarlPenguin replied:
done, corrected

wiktorek140 commented on config.fs:
```diff
@@ -1,155 +1,161 @@
+[AID_VENDOR_RFS_SHARED]
+value:2952
+
+[system/vendor/bin/wcnss_filter]
```
Should be system/vendor/ or vendor/ ?

JarlPenguin replied:
fixed

wiktorek140 commented on configs/privapp-permissions-hotword.xml:
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<permissions>
+    <privapp-permissions package="com.android.hotwordenrollment.okgoogle">
+        <permission name="android.permission.MANAGE_VOICE_KEYPHRASES"/>
+    </privapp-permissions>
+
+    <privapp-permissions package="com.android.hotwordenrollment.tgoogle">
+        <permission name="android.permission.MANAGE_VOICE_KEYPHRASES"/>
+    </privapp-permissions>
+
+    <privapp-permissions package="com.android.hotwordenrollment.xgoogle">
+        <permission name="android.permission.MANAGE_VOICE_KEYPHRASES"/>
+    </privapp-permissions>
+</permissions>
```
It is even required?

wiktorek140 commented on gps/android/Gnss.h:
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2017-2018-2018-2018, The Linux Foundation. All rights reserved.
```
why 2018 three times?

JarlPenguin replied:
done

wiktorek140 commented on manifest.xml:
```diff
@@ -111,12 +124,13 @@
         <interface>
             <name>IGnss</name>
             <instance>default</instance>
+            <fqname>@1.0::IGnss/gnss_vendor</fqname>
```
1.0 or 1.1 ?

JarlPenguin replied:
addressed, fqname should be 1.0

wiktorek140 commented on msm8937.mk:
```diff
@@ -82,37 +82,48 @@ PRODUCT_PACKAGES += \
     android.hardware.audio@2.0-service \
-    android.hardware.audio.effect@2.0-impl \
+    android.hardware.audio.effect@4.0-impl \
+    android.hardware.audio.effect@2.0-service \
```
Why service 2.0?

JarlPenguin replied:
impl is 4.0, service is 2.0, idk why
but it's like that everywhere else, I'm assuming a 4.0-service doesn't exist

wiktorek140 commented on msm8937.mk:
```diff
@@ -132,11 +143,18 @@ PRODUCT_PACKAGES += \
+
+#PRODUCT_COPY_FILES += \
+#    $(LOCAL_PATH)/configs/qdcm_calib_data_mipi_mot_vid_inx_1080p_497.xml:$(TARGET_COPY_OUT_VENDOR)/etc/qdcm_calib_data_mipi_mot_vid_inx_1080p_497.xml \
+#    $(LOCAL_PATH)/configs/qdcm_calib_data_mipi_mot_vid_tianma_1080p_497.xml:$(TARGET_COPY_OUT_VENDOR)/etc/qdcm_calib_data_mipi_mot_vid_tianma_1080p_497.xml
```
Extract it to montana tree

JarlPenguin replied:
removed, we don't have these anyways

wiktorek140 commented on msm8937.mk:
```diff
@@ -266,15 +293,20 @@ PRODUCT_PACKAGES += \
 # Permissions
 PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/configs/privapp-permissions-oem.xml:$(TARGET_COPY_OUT_SYSTEM)/etc/permissions/privapp-permissions-oem.xml \
+    $(LOCAL_PATH)/configs/privapp-permissions-hotword.xml:$(TARGET_COPY_OUT_SYSTEM)/etc/permissions/privapp-permissions-hotword.xml
```
It is required?

JarlPenguin replied:
fpushed out of branch

JarlPenguin committed:
msm8937-common: remove references of display calibration configs

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: move permissions under a single label

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I77de99ae4d72e2810aa7c2f6ef91145c1cf28c98

JarlPenguin committed:
msm8937-common: update check_features for P

* Move it to /system, because we don't need it after it is executed

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ia0f8214b36da3b0d68cbe2b31438c619cd20bc3a

JarlPenguin committed:
msm8937-common: generate Treble-compatible makefiles

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ie11e1fc79c984650b6102886aa788e57ac178b58

wiktorek140 commented on rootdir/etc/ueventd.qcom.rc:
```diff
@@ -326,3 +327,8 @@
 #Egistec fingerprint
 /dev/esfp0		  0660   system     system
+
+# USB role switch
+/sys/class/dual_role_usb/*	data_role	0444	system	system
+/sys/class/dual_role_usb/*	power_role	0444	system	system
+/sys/class/dual_role_usb/*	mode		0444	system	system
```
required for pie? or required for all devices?

wiktorek140 commented on sepolicy/file_contexts:
```diff
@@ -112,4 +107,14 @@
+/sys/devices/virtual/timed_output/vibrator/vtg_default                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_level                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_min                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_max                            u:object_r:sysfs_vibrator:s0
```
Required for vibration?

JarlPenguin replied:
for the new HAL it seems like it is required

wiktorek140 commented on system_prop.mk:
```diff
@@ -41,6 +41,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_PROPERTY_OVERRIDES += \
     bluetooth.hfp.client=1 \
     qcom.bluetooth.soc=smd \
+    vendor.qcom.bluetooth.soc=smd \
```
Looks like doubled. Should be both?

JarlPenguin replied:
not doubled

wiktorek140 commented on thermal/thermal_common.h:
```diff
@@ -0,0 +1,73 @@
+#include <hardware/thermal.h>
+#define ARRAY_SIZE(x) (int)(sizeof(x)/sizeof(x[0]))
+
+enum therm_msm_id {
+    THERM_MSM_UNKNOWN = 0,
+    THERM_MSM_8953,
+    THERM_SDM_660,
+    THERM_SDM_630,
+    THERM_SDM_710,
+    THERM_QCS_605,
+    THERM_SDM_632,
+    THERM_SDM_439,
+    THERM_MSMNILE,
+    THERM_TALOS,
+    THERM_SDMMAGPIE,
+    THERM_MSM_8917,
+    THERM_TRINKET,
+    THERM_KONA,
+    THERM_LITO,
+    THERM_ATOLL,
+};
```
I dont see MSM_8937. or we use 8953?

JarlPenguin replied:
most likely sdm439, since it inherits from msm8937

Sohamlad7 committed:
msm8937-common: init: Remove useless platform check

justks committed:
msm8937-common: cleanup fqnames for DRM

Change-Id: Ic164e57d8aec4872e505abe138095a338f0ee85a

JarlPenguin committed:
msm8937-common: cleanup firmware symlinks

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If1c51f8cdecdf1a95fdf90192c0b44739edfbde8

JarlPenguin committed:
msm8937-common: kang missing bluetooth HAL from sanders

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I8288b49bacfa4b1fe9b9086fb3700bbf6c08c6cc

JarlPenguin committed:
msm8937-common: sepolicy: cleanup for P

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: thermal: sync with LA.UM.7.6.r1-06400-89xx.0

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I637efc0ff60810b9f2a2fa47aee4a0b0c53107f1

JarlPenguin committed:
msm8937-common: remove outdated packages

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I6a085eccf3ba0ec6bfc975fc9e29b9fbe1a3ac12

JarlPenguin committed:
Revert "msm8937-common: enforce priv-app permissions"

* It's enforced by default now, so no need to set the prop.

This reverts commit 1cb67e3154bcc12f710217ddd28c48ebba8b8a7a.

JarlPenguin committed:
msm8937-common: rootdir: remove esepmdaemon

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ib6b296bcb1498594681b45b3b0166bed8419c915

JarlPenguin committed:
Revert "msm8937-common: move wcnss-service to late_start"

* Not needed anymore with a4063406d1a7e605e806e9b32d5318584efb1075.

This reverts commit c0699076ace7833ec35ab35c24bd3d483bdc4ed8.

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I19e36e2734bb33553985b9fff8052489789d48f7

JarlPenguin committed:
msm8937-common: rootdir: disable 2nd RIL daemon by default

* Start it only when we are dual SIM, since we don't need it on single SIM models

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Id7dc106270b2437ea1d87a3c9c866424fcd089e1

JarlPenguin committed:
msm8937-common: cleanup BoardConfig headers

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If8af433414cf3e758753ab4559cc5efb6b85cceb

JarlPenguin committed:
msm8937-common: regenerate manifest

* lshal --init-vintf

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I1cfc0896a2996d1efee9e9945362b5aa30e74e12

JarlPenguin committed:
msm8937-common: update RIL lib path

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I99889715e85c6040d46d8be3abcc84071d881a39

JarlPenguin commented:
-2, wifi dead

CypherNoodle committed:
msm8937-common: remove duplicate packages

JarlPenguin committed:
msm8937-common: drop NFC entries to device specific manifests

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: build clearkey DRM service from source

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: do not build some packages by default

* Not every msm8937 Motorola device has a front mounted fingerprint sensor, so don't build Touch HAL by default
* Also do not define navigation bar overlay, because Touch HAL will not function properly when it's set to true
* Decommonize LineageActions overlay, not every device uses
LineageActions as KeyHandler

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: create root folders at build time

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: move some frameworks to vendor

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
[DNM][TEMP] msm8937-common: Disable SELinux

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
[DNM][TEMP] msm8937-common: build vintf

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: add framework manifest

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

Sohamlad7 committed:
msm8937-common: rootdir: Change MTP parameters

Set correct USB controller configuration like mtp_rx_len, mtp_tx_len
for msm8937. This fixes MTP

mtp_rx_req_len : 524288 to 1048576
mtp_tx_reqs : 524288 to 16384

JarlPenguin committed:
msm8937-common: check_features: cleanup

* Nuke symlink checks, they didn't work properly anyways
* Also mount /vendor beforehand in releasetools

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: update some camera props

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: drop touch HAL entry to device manifest

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: uprev vendor.display.color to 1.2

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: check_features: fix NFC removal

* Move /vendor mounting to script, since in releasetools it gives a build error

Co-authored-by: The_Baztyx <11617389+CypherNoodle@users.noreply.github.com>

JarlPenguin committed:
msm8937-common: check_features: remove compass for Brazilian models

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: rootdir: move USB to init.mmi.usb.rc

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: vintf is no longer needed

This reverts commit f5e141e37e7f10aeb1ab69269fb2c8dd3ea721b8.

JarlPenguin committed:
msm8937-common: Build SEPolicy

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

JarlPenguin committed:
msm8937-common: sepolicy: write from scratch

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: Ibfee18725182cd1cb382b1b416184d621bac9bd3

JarlPenguin committed:
msm8937-common: enable VNDK runtime isolation

* Build system variants of HIDL blobs
* Decommonize lib_mottof.so
* Move IMS blobs to system
* Nuke 32 bit variants of IMS blobs, they're not used

E linker  : library "/system/priv-app/ims/lib/arm64/libimsmedia_jni.so" ("/vendor/lib64/libimsmedia_jni.so") needed or dlopened by "/system/lib64/libnativeloader.so" is not accessible for the namespace: [name="classloader-namespace", ld_library_paths="", default_library_paths="/system/priv-app/ims/lib/arm64:/system/priv-app/ims/ims.apk!/lib/arm64-v8a:/system/lib64:/system/lib64", permitted_paths="/data:/mnt/expand:/data/data/org.codeaurora.ims:/system/priv-app/ims:/system/lib64:/system/lib64/drm:/system/lib64/extractors:/system/lib64/hw:/system/product/lib64:/system/framework:/system/app:/system/priv-app:/vendor/framework:/vendor/app:/vendor/priv-app:/odm/framework:/odm/app:/odm/priv-app:/oem/app:/system/product/framework:/system/product/app:/system/product/priv-app:/data:/mnt/expand"]
E: AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: library "android.hidl.base@1.0.so" not found

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: If9182b1834604f6b37fb96eff760945aabf34da9

JarlPenguin committed:
msm8937-common: disable world phone bool

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ifa85c111259500953a6984367c763baf4edc98f2

JarlPenguin committed:
msm8937-common: shim libmmcamera_ppeiscore.so

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I2c4900e24d3879b1927641e16477d64989adad42

erfanoabdi committed:
msm8937-common: Update audio and ADSP blobs

Change-Id: I968cc6821d41e44faeaeb2f4007844b87d6dc9b7

JarlPenguin committed:
msm8937-common: Update audio configs from CAF

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I6d1a97992edb0f460a31e4f70d8edce8c988824d

JarlPenguin committed:
Revert "msm8937-common: uprev Thermal hals to 1.1"

This reverts commit aadb947bf75a019065d69c6191c642e6f6ab7a93.

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ib26851c15189de83bc7737e864ac8efbe85f2dc2

JarlPenguin committed:
msm8937-common: Remove sound trigger configs

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: I4f53204309ed6fb4f17c8731ecebf90ac821697b

JarlPenguin committed:
msm8937-common: Drop unused Bluetooth blobs

* Also remove unused props and packages

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>

erfanoabdi committed:
msm8937-common: Update time service blobs

* From deen PPKS29.68-16-36-13

Signed-off-by: JarlPenguin <jarlpenguin@outlook.com>
Change-Id: I49ac674e783b4bb740cbdbd5d59708b7445d75e5

JarlPenguin committed:
msm8937-common: Switch to LineageOS dtbtool

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ife9833bbe75dccc8720b540538130dc374035433

JarlPenguin committed:
msm8937-common: Set kernel offsets

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Ibf336ebf40b1ebe0a30443d3f6684bf92f36af89

erfanoabdi committed:
msm8937-common: Update RIL blobs

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: Id029ced358d0779720e53e1b8d3c177000f00a07

JarlPenguin committed:
msm8937-common: Build libjson from source

Signed-off-by: Jarl-Penguin <jarlpenguin@outlook.com>
Change-Id: I706a71a9328fa2b61ff3abc2e3be1cc88a720200
